changelogs/unreleased/-1x68xu9.yml

```diff
@@ -0,0 +1,14 @@
+---
+title: Improved: code to prepare loader on app mounted and assign it to null on dismiss
+ticket_id: "#1x68xu9"
+merge_request: 2
+author: Yashi Verma
+type: changed
+
```

Comment:
Please remove these unwanted empty lines

Reply:
Removed the unwanted lines